src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,35 @@
+import PropTypes from 'prop-types';
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = (props) => {
+  const likeButtonClicked = () => {
+    props.likeToggleFunc(props.id);
+  };
 
-const ChatEntry = () => {
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={props.senderType === 'local' ? 'chat-entry local' : 'chat-entry remote'}>
```

Comment:
You could also assign this ternary logic to a variable before your `return` statement for readability. However, note that you have very similar logic here and in `ChatLog.jsx`. How could you refactor this logic to DRY up your components? @omatesh